todocore/src/main/java/com/knighttodo/todocore/rest/request/DayTodoRequestDto.java

```diff
@@ -16,6 +16,9 @@
 @Builder
 public class DayTodoRequestDto {
 
+    @NotNull
+    private int orderNumber;
```

Comment:
primitive value can't be Null, so @NotNull annotation looks redundant for me.

Reply:
Thanks.